arbnode/seq_coordinator.go

```diff
@@ -495,7 +526,7 @@ func (c *SeqCoordinator) update(ctx context.Context) time.Duration {
 			log.Error("myurl main sequencer, but no sequencer exists")
 			return c.noRedisError()
 		}
-		err := c.chosenOneUpdate(ctx, localMsgCount, localMsgCount, nil)
+		err := c.chosenOneUpdate(ctx, localMsgCount, localMsgCount, nil, false)
```

Comment:
This is the only place a secondary sequencer can become the main one. Logic for force-including delayed messages belongs here.